### PR TITLE
Added Lombok dependency

### PR DESCRIPTION
**Pull request by anandjakhaniya**

### CODE REVIEW

File: backend/common/src/main/java/ai/verta/modeldb/common/config/ArtifactStoreConfig.java [hunk 1]
```diff
@@ -1,40 +1,56 @@
...
+import lombok.Data;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter(AccessLevel.PRIVATE)
```
Comment (jkwatson-verta) [inline backend/common/src/main/java/ai/verta/modeldb/common/config/ArtifactStoreConfig.java]: can we get away with `AccessLevel.NONE` ?

File: backend/common/src/main/java/ai/verta/modeldb/common/configuration/EnabledMigration.java [hunk 2]
```diff
@@ -19,7 +19,7 @@ private static boolean isMigration() {
             Optional.ofNullable(System.getenv(CommonConstants.ENABLE_LIQUIBASE_MIGRATION_ENV_VAR))
                 .orElse("false"));
     LOGGER.info("ENABLE_LIQUIBASE_MIGRATION_ENV_VAR: {}", enableLiquibaseMigrationEnvVar);
-    return enableLiquibaseMigrationEnvVar;
+    return true;
```
Comment (jkwatson-verta) [inline backend/common/src/main/java/ai/verta/modeldb/common/configuration/EnabledMigration.java]: ??? This change seems very wrong.

**Review by jkwatson-verta — DISMISSED** [review]

File: backend/common/src/main/java/ai/verta/modeldb/common/configuration/EnabledMigration.java (hunk 2 above)
Comment (anandjakhaniya) [reply backend/common/src/main/java/ai/verta/modeldb/common/configuration/EnabledMigration.java]: It was committed by mistake. It's just my hack while testing and forgot to revert back. 😇

Comment (jkwatson-verta) [inline backend/common/src/main/java/ai/verta/modeldb/common/configuration/EnabledMigration.java]:
```suggestion
    return enableLiquibaseMigrationEnvVar;
```

File: backend/common/src/main/java/ai/verta/modeldb/common/config/ArtifactStoreConfig.java (hunk 1 above)
Comment (anandjakhaniya) [reply backend/common/src/main/java/ai/verta/modeldb/common/config/ArtifactStoreConfig.java]: Updated code with `AccessLevel.NONE`.

**Review by jkwatson-verta — APPROVED** [review]

**Pull request merged**